### DISCUSSION TITLE
Support for mpd satellite mode.

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#275] [Feature Requests]
I have a server in my house already running mpd. It would be nice if rAudio-1 could be set up as an mpd satellite server. That would significantly speed up scan times.

**Comment by RafaelEstevamReis** [comment]
As I understand (i'm new at rAudio inner workings) will not be easy
rAudio calls mpd executable directly (not tcp commands) for some functions, and control externally others functions like PlayLists

This is an old choice (maybe "RuneAudio" old) seems to me that RuneAudio starts as a standalone and then added mpd to some stuff, and then mpd grow in _some_ areas
